PluginBuilder/Controllers/ApiController.cs

```diff
@@ -106,24 +108,73 @@ public async Task<IActionResult> Download(
         [ModelBinder(typeof(PluginVersionModelBinder))] PluginVersion version)
     {
         await using var conn = await ConnectionFactory.Open();
-        var url = await conn.ExecuteScalarAsync<string?>(
-            "SELECT b.build_info->>'url' FROM versions v " +
+
+        var result = await conn.QueryFirstOrDefaultAsync<(string? Url, long BuildId)>(
+            "SELECT b.build_info->>'url' AS Url, v.build_id AS BuildId " +
+            "FROM versions v " +
             "JOIN builds b ON b.plugin_slug = v.plugin_slug AND b.id = v.build_id " +
             "WHERE v.plugin_slug=@plugin_slug AND v.ver=@version",
             new
             {
                 plugin_slug = pluginSlug.ToString(),
                 version = version.VersionParts
             });
-        if (url is null)
+
+        if (result == default || result.Url is null)
```

Comment:
A tuple can't be null, so you can just replace this by `result.Url is null` only.

Reply:
This is updated